cmd/dlv/main.go

```diff
@@ -28,6 +30,14 @@ var (
 )
 
 func main() {
+
+	// Config setup and load.
+	err := config.CreateConfigPath()
+	if err != nil {
+		fmt.Printf("Could not create config directory: %v.")
+	}
+	conf := config.LoadConfig()
```

Comment:
Sounds great to me. They are separate at the moment because they were in different places before, with CreateConfigPath existing before I added this new stuff. It makes sense to consolidate them now.